crates/rpc/src/v02/method/get_class.rs

```diff
@@ -17,15 +16,14 @@ pub async fn get_class(
     context: RpcContext,
     input: GetClassInput,
 ) -> Result<ContractClass, GetClassError> {
-    let block = match input.block_id {
+    let (block_id, is_pending) = match input.block_id {
         BlockId::Pending => {
-            if is_pending_class(&context.pending_data, input.class_hash).await {
-                BlockId::Pending
-            } else {
-                BlockId::Latest
-            }
+            let latest = pathfinder_storage::BlockId::Latest;
+            let is_pending = is_pending_class(&context.pending_data, input.class_hash).await;
+            (latest, is_pending)
         }
-        other => other,
+        // unwrap is safe as only pending fails to cast.
+        other => (other.try_into().unwrap(), false),
```

Comment:
nit: consistency with other methods
```suggestion
        other => (other.try_into().expect("Only pending cast should fail"), false),
```